react/features/base/connection/actions.native.js

```diff
@@ -68,6 +69,29 @@ export type ConnectionFailedError = {
     recoverable?: boolean
 };
 
+/**
+ * Type of parameter got from native app in an {@code NativeXmppPostMethodEventData}.
+ *
+ * In some cases, functions take callback methods as parameter.
+ * eg. connection.muc.join(roomId, nick, onMessage: () => boolean, onPresence: () => boolean)
+ * In this case we send a specific object from native app,
+ * eg. { nativeResponseType: 'chat_message', defaultReturnValueOfCallback: true }
+ * We have to call post method with a callback method, and return the value to native app with which callback is called,
+ * to let it to handle on its own.
+ */
+type NativeXmppPostMethodEventParam = any | { nativeResponseType: string, defaultReturnValueOfCallback?: any };
```

Comment:
`any | WhateverOtherType` is kind of a silly type definition. This could just be anything.

Reply:
I think it is helpful as documentation, from native we can get any object as param, but if it contains nativeResponseType property, we should handle that param different way.